test/assert_meta_test.exs

```diff
@@ -0,0 +1,49 @@
+defmodule AssertMetaTest do
+  use ExUnit.Case
+  import JsonApiAssert, only: [assert_meta: 2, assert_meta: 1]
+  import JsonApiAssert.TestData, only: [data: 1]
+
+  @meta %{
+    "meta" => %{
+      "license" => "The MIT License (MIT)",
+      "authors" => [
+        "Brian Cardarella"
+      ]
+    }
+  }
+
+  test "assert will return original payload" do
+    assert Map.merge(data(:payload), @meta) ==
```

Comment:
can you break these results into two variable so the compare is on the same line?

```elixir
expected = Map.merge(data(:payload), @meta)
result = assert_meta(...)

assert result == expected
```